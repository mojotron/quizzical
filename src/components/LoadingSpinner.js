import "../styles/LoadingSpinner.css";
const LoadingSpinner = () => {
  // source https://loading.io
  return (
    <div className="lds-grid">
      <div></div>
      <div></div>
      <div></div>
      <div></div>
      <div></div>
      <div></div>
      <div></div>
      <div></div>
      <div></div>
    </div>
  );
};

export default LoadingSpinner;
