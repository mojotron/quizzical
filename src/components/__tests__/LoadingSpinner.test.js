import { render } from "@testing-library/react";
import LoadingSpinner from "../LoadingSpinner";

describe("LoadingSpinner component", () => {
  test("render snapshot", () => {
    const { component } = render(<LoadingSpinner />);
    expect(component).toMatchSnapshot();
  });
});
