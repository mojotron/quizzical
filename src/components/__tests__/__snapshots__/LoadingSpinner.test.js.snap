// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingSpinner component render snapshot 1`] = `undefined`;
